tests/kuttl/tests/deployments/01-deploy-ironic.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
   - script: |
-      cp ../../../../config/samples/ironic_v1beta1_ironic.yaml deploy
-      oc kustomize deploy | oc apply -n openstack -f -
+      cp ../../../../config/samples/ironic_v1beta1_ironic.yaml deploy1
```

Comment:
I would suggest renaming the `deploy1` and `deploy2` folders to something more explicity like `deploy_ironic` and `deploy_ironic_conductor`